data/time_series.go

```diff
@@ -143,11 +208,18 @@ func LongToWide(longFrame *Frame) (*Frame, error) {
 		if currentTime.After(lastTime) { // time advance means new row in wideFrame
 			wideFrameRowCounter++
 			lastTime = currentTime
-			for _, field := range wideFrame.Fields {
+			for wideFrameIdx, field := range wideFrame.Fields {
 				// extend all wideFrame Field Vectors for new row. If no value found, it will have zero value
 				field.Extend(1)
+				if wideFrameIdx == 0 {
+					wideFrame.Set(wideFrameIdx, wideFrameRowCounter, currentTime)
+					continue
+				}
+				fillVal, err := getMissing(fillMissing, field, wideFrameRowCounter)
+				if err == nil {
```

Comment:
Just checking, we should suppress any error here, i.e. not propagate it?

Reply:
`wideFrame.Set` actually overwrites the zero value of the field type set by [`field.Extend`](https://github.com/grafana/grafana-plugin-sdk-go/blob/198aced61e13ed18652bd465167115532907e316/data/time_series.go#L213)
In case of error (fillMissing is disabled) the zero value is kept.